drivers/clock_control/clock_stm32f0_f3.c

```diff
@@ -77,6 +77,11 @@ void config_enable_default_clocks(void)
 	/* Enable System Configuration Controller clock. */
 	LL_APB1_GRP2_EnableClock(LL_APB1_GRP2_PERIPH_SYSCFG);
 #endif
+#else
+#if defined(CONFIG_USB_DC_STM32)
```

Comment:
I thought it was done already in `gpio_stm32_enable_int`. I wonder why this doesn't work...

Reply:
You are right. I missed that it is already enabled in `gpio_stm32_enable_int`. Initialization priority seems fine, it is strange that it is not enabled.